include/tesla.hpp

```diff
@@ -193,6 +189,48 @@ namespace tsl {
     // Helpers
 
     namespace hlp {
+
+        /**
+         * @brief Capture the whole screen with overlays
+         * 
+         */
+        static void captureScreen() {
```

Comment:
Formatting in this function looks horrible lol
Try to follow the style of the rest of libtesla please like in the rest of your PR :P